### DISCUSSION TITLE
Snes multitap on Super Bomberman 2

### DISCUSSION THREAD

**Discussion by cubeyz** [darthcloud/BlueRetro#769] [Q&A]
Hello, I recently got a blueretro for my Snes but I can't get multitap to work right. I selected slot 2 but even when I connect multiple controllers to it it doesn't seem to work. 
I updated the firmware to 1.8.4 but it hasn't helped. It just let's me use one controller as player 2.

**Comment by darthcloud** [comment]
Unless you build it yourself, it's unlikely that what you got have the proper wiring for multitap support.

**Reply by cubeyz** [reply]
I guess that makes sense. I just thought as it was in the software as an option and let multiple controllers connect it would work.
